string[] gainedSupportedMutations
RequiredMutation[] requiredMutationsRequiredForNextPhase
---
CandidateModules[] candidates
---
